source/process/community-process.rst

```diff
@@ -92,47 +90,28 @@ The various community channels are outlined below, along with the responsibiliti
 
       - https://github.com/mattermost/mattermost-server/issues
       - https://github.com/mattermost/docs/issues
-      - https://github.com/mattermost/ios/issues
-      - https://github.com/mattermost/push-proxy/pulls
-      - https://github.com/mattermost/mattermost-docker-preview/issues
-      - https://github.com/mattermost/mattermost-bot-sample-golang/issues
-      - https://github.com/mattermost/mattermost-load-test/issues
       - https://github.com/mattermost/mattermost-api-reference/issues
+      - https://github.com/mattermost/mattermost-bot-sample-golang/issues 
+      - https://github.com/mattermost/mattermost-load-test/issues
+      - https://github.com/mattermost/mattermost-push-proxy
       - [a month or older - reviews are monthly] https://github.com/mattermost/android/pulls
       - [a month or older - reviews are monthly] https://github.com/mattermost/ios/pulls
-
-  - Community PM Responsibility:
-
-    - Review the list and either reply (with help from dev if needed) or route to the appropriate person, following up to make sure a response is posted
+      - http://stackoverflow.com/questions/tagged/mattermost?sort=newest&pageSize=15 
+      - https://gitlab.com/search?utf8=%E2%9C%93&search=mattermost&group_id=&project_id=20699&scope=issues&repository_ref= 
+      - https://gitlab.com/gitlab-org/gitlab-mattermost/issues 
+      - https://gitlab.com/gitlab-org/omnibus-gitlab/issues?label_name=Mattermost
+      - https://gitlab.com/gitlab-org/gitlab-ce/issues?label_name=mattermost
 
 4. Help Wanted GitHub Issues:
 
  - Description:
 
-   - New GitHub Help Wanted issues are automatically created from JIRA tickets and are open for community contributions
-
- - Community PM Responsibility:
-
-   - Review newly created Help Wanted issues for formatting and apply difficulty and language labels defined in the issue description by the triage team
-   - Respond to questions from community members, which get automatically posted in the `Community Heartbeat channel <https://pre-release.mattermost.com/core/channels/community-heartbeat>`_
-   - Follow up with a community member if there hasn't been a response for more than two weeks
-
-5. Customer Zendesk Issues:
-
-  - Description:
-
-    - Sales team will @mention the community PM in Mattermost with the Zendesk ticket that needs a response
-
-  - Community PM Responsibility:
-
-    - Click the Zendesk hashtag to review previous ticket conversation (or view directly in Zendesk)
-    - In the Mattermost thread: Answer the question (with dev help if necessary), or draft a response asking follow up questions
-    - @mention Hanna, and she will reply to the customer in Zendesk
+   - New GitHub Help Wanted issues are automatically created from JIRA tickets and are open for community contributions.  The Community Manager can disregard these issues as they do not need to be posted to the Peer-to-peer Help channel.
```

Comment:
Do we need to check Help Wanted issues every so often to see if there's comments on them that no one has responded to? Or is that covered by PMs?

Reply:
Process to date has been that this is PMs responsibility...  Let me know if you'd like to perhaps add a monthly sweep by Community Manager?

Reply:
Just happened to see this comment - I'm happy to take on checking Help Wanted issues regularly if that sounds good, unless otherwise decided.

Reply:
Thanks @amyblais :)

Let me know what you decide @lfbrock and I'll update the PR if you'd prefer community manager to check 'help wanted' issues...

Reply:
@lindy65 - I think Amy checking will cover it, I also did a check of existing help wanted issues and there were only about 3 that didn't receive a response from us (mostly from around Christmas holidays), so I think Amy or other PMs checking periodically will cover it.

Reply:
@lindy65 - I think Amy checking will cover it, I also did a check of existing help wanted issues and there were only about 3 that didn't receive a response from us (mostly from around Christmas holidays), so I think Amy or other PMs checking periodically will cover it.

Reply:
All good, thanks @lfbrock @amyblais 

I'll get this merged as is then